common-test/src/main/java/io/customer/commontest/BaseTest.kt

```diff
@@ -1,10 +1,10 @@
-package io.customer.common_test
+package io.customer.commontest
```

Comment:
According to Kotlin's recommendation, `_` can be repaced by either the single word `commontest` or `commonTest`. 

Maybe we should set criteria of when to use when? so there is not confusion?